src/Evolution/Executables/GeneralizedHarmonic/EvolveGeneralizedHarmonic.hpp

```diff
@@ -156,14 +156,17 @@ struct EvolutionMetavars {
   using analytic_solution_fields =
       db::get_variables_tags_list<typename system::variables_tag>;
   using observe_fields = tmpl::append<
-      analytic_solution_fields,
-      tmpl::list<
+      tmpl::push_back<
```

Comment:
Please file an issue to fix the four-index constraint and constraint_energy to either work in all dimensions or remove the dimension template parameter...

Reply:
#2223